geopandas/tests/test_sindex.py

```diff
@@ -19,6 +19,13 @@
 import pytest
 from numpy.testing import assert_array_equal
 
+try:
+    from scipy.sparse import coo_array  # noqa: F401
+
+    HAS_SCIPY = True
+except ImportError:
+    HAS_SCIPY = False
+
```

Comment:
```suggestion
SCIPY_MARK = pytest.mark.skipif(not HAS_SCIPY, reason="scipy not installed")
```
this could make the test code a little cleaner